### DISCUSSION TITLE
Quarkus update command rewrites unrelated dependencies

### DISCUSSION THREAD

**Discussion by siewp** [quarkusio/quarkus#44528] [Community]
Hello there,
while the Quarkus update command is very useful when it rewrites stuff related to Quarkus and all its extensions, it seems it is also rewriting dependencies in the `pom.xml` of a Quarkus project which are not directly related to the Quarkus ecosystem. An example would be the Protobuf dependency which we might use in our projects with a different version than Quarkus (probably used by the Quarkus gRPC extension which **we do not use**). In particular, the update command removes the version from the `pom.xml` of our Quarkus projects since it is present in the Quarkus BOM although we would like to manage this version by ourselves. 

Is there a way to disable this behavior, such that rewriting is done for Quarkus dependencies, but other dependencies like Protobuf are not affected? We would like to use the update command to further automate our update processes, but the removal of versions from the `pom.xml` always requires manual action.

**Comment by gsmet** [comment]
Hi!

This behavior is indeed not intended. Could you open an issue with a reproducer (a simple Maven project I can try to update to see the issue you're mentioning)?

Thanks!

**Reply by siewp** [reply]
Thanks for the quick reply! Yes, I will create an issue with a reproducer.

**Reply by gsmet** [reply]
Awesome, thanks! My guess is that it's OpenRewrite doing too much work but I need to figure out why and if we can improve on that.

**Reply by siewp** [reply]
Here you go: https://github.com/quarkusio/quarkus/issues/44529

**Reply by gsmet** [reply]
Thanks, I'll have a look next week and see if we can improve or if we need to report it to OpenRewrite.
